### ISSUE TITLE
Blur image classification accuracy does not increase.

### ISSUE THREAD

**Issue by kimsjpk1** [aditya9211/Blur-and-Clear-Classification#7]
Hello? I'm working on blur image classification using the network on this site.
I have 2,500 images, and I have blur image and clean image for classification, but when I train,
the validation dataset accuracy is not more than 70% and I have a stagnant problem
Please tell us how can improve our validation accuracy. and I want to know the hyper parameter setting
for your neural network you set.
Thank you

**Comment by aditya9211** [comment]
Hi,
Sorry for late response.
For the above problem, Can you check the skewness if exits in data? 
As in that case, accuracy will not be a good metric for the clear picture
If all good keep an eye on training accuracy also, if the case of overfitting/underfitting occurs and always recheck by increasing data for better results.

For hyper_parameters part kindly have a look to *config.py* file in repo
If still any queries respond back

**Comment by kimsjpk1** [comment]
p{margin-top:0;margin-bottom:0}
    


    Thanks for your mail.

I already solve a blur image classification using the following method.

"A shallow convolutional neural network for blind image sharpness assessment"

This is a useful way so you should note please.

--------- 원본 메일 ---------

    
        보낸사람: Aditya Sharma <notifications@github.com>
        받는사람: aditya9211/Blur-and-Clear-Classification <Blur-and-Clear-Classification@noreply.github.com>
        
            참조: kimsjpk1 <kimsjpk@hanmail.net>, Author <author@noreply.github.com>
        
        날짜: 18.11.24 18:27 GMT +0900
        제목: Re: [aditya9211/Blur-and-Clear-Classification] Blur image classification accuracy does not increase. (#7)
        
        
    
    
        Hi kimsjpk1,
Is the issue got resolved, or the same stagnant problem?
Aditya

**Comment by aditya9211** [comment]
Hi @kimsjpk1 
Thanks for the suggestion, *Will look upon this paper to implement it and  make it better*

Aditya
